ado/enforce-clang-format.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
+clang-format -style=file -i stl/inc/** stl/src/** 2>&1
+echo If your build fails here, the following files were changed by clang-format.

```

Comment:
Perhaps "... you need to clang-format the following files and resubmit"?

Reply:
That's clearer, yes.